lib/migration.pm

```diff
@@ -143,6 +144,13 @@ sub disable_installation_repos {
     }
 }
 
+# Based on bsc#1097111, need to disable kernel multiversion before migration, and enable it after migration
+# https://documentation.suse.com/sles/15-SP3/html/SLES-all/cha-update-preparation.html#sec-update-preparation-multiversion 
+sub disable_kernel_multiversion_temporarily {
```

Comment:
I think you can just name the function as disable_kernel_multiversion, simple and easy. :)